sprout/templates/project-id-metadata-view.html

```diff
@@ -41,18 +68,32 @@
 
   function rowSelected(row) {
     // Deselect all other rows
-    document.querySelectorAll('.selectable').forEach(function (otherRow) {
-      otherRow.classList.remove('selected');
+    document.querySelectorAll(".selectable").forEach(function (otherRow) {
+      otherRow.classList.remove("selected");
     });
 
     // Add selected class
-    row.classList.add('selected');
+    row.classList.add("selected");
 
     // Update the href and disabled state of the edit and upload buttons based on row selection
-    document.querySelector("#edit-link").href = edit_url.replace('0', row.id);
+    document.querySelector("#edit-link").href = edit_url.replace("0", row.id);
     document.querySelector("#edit-link").removeAttribute("disabled");
-    document.querySelector("#upload-link").href = upload_url.replace('0', row.id);
+    document.querySelector("#upload-link").href = upload_url.replace("0", row.id);
     document.querySelector("#upload-link").removeAttribute("disabled");
+
+    // Get all metadata columns
+    let metadata_columns = document.getElementsByClassName("metadata_columns_table");
+
+    // Loop through all the metadata columns and show the ones that matches the selected row
+    for (let column_number = 0; column_number < metadata_columns.length; column_number++) {
+      // If the metadata column's id matches the selected row's id, remove the 'hide' class
+      if (metadata_columns[column_number].id === row.id) {
+        metadata_columns[column_number].classList.remove("hide");
+      } else {
+        // Otherwise, add the "hide" class
+        metadata_columns[column_number].classList.add("hide");
+      }
+    }
```

Comment:
Small thing. Maybe you can use this for-loop instead (This is not supported in IE, but that is okay. I have already done many things not supported by IE )

```suggestion
    // Loop through all the metadata columns and show the ones that matches the selected row
    for (const metadata_column of metadata_columns){
      // If the metadata column's id matches the selected row's id, remove the 'hide' class
      if(metadata_column.id === row.id){
        metadata_column.classList.remove("hide")
      } else {
        // Otherwise, add the "hide" class
        metadata_column.classList.add("hide")
      }
    }
```

Reply:
@philter87 Ah, alright! Is the one I implemented an older version of a for-loop or how come there's different kinds?